### ISSUE TITLE
Add a button to return to the dispute tools from the messages screen

### ISSUE THREAD

**Issue by marcondag** [debtcollective/parent#200]
Replaces https://github.com/debtcollective/parent/issues/198

Will eventually be superseded by https://github.com/debtcollective/parent/issues/139 which would implement the header with a link to the admin backend for the tools when an administrator is logged in.

Two potential solutions for fixing this:

1. Adding a button to the messages page specifically to link the admin back to the disputes back office (should only be visible to administrators)
2. Adding a similar button to the header, just as a holdover until the themed header is implemented in discourse

Another question, for members, should we go ahead and add a button that goes back to the "my disputes" page from the messages in the community?

### Acceptance Criteria

**Needs to be refined!**

- [ ] There is an easy way for administrators to return to the dispute tools back office from the messages page

**Comment by dawn3296** [comment]
Another question, for members, should we go ahead and add a button that goes back to the "my disputes" page from the messages in the community?

Yes! That would be really awesome.

**Comment by marcondag** [comment]
We should definitely remove that text area on the dispute tracking screen. Instead we could add a link to the private message thread for that dispute. I'll add this to the acceptance criteria for the other ticket that changes the messages to be single threaded as that work will need to be done first before we can link the member directly to the thread.

**Comment by orlando** [comment]
We have this ticket to add the links to the Discourse menu https://github.com/debtcollective/parent/issues/139, We will only add the links since replacing the whole Discourse header is not an option
